src/sbr/easinTemplates.py

```diff
@@ -40,10 +40,10 @@
 spring:
   profiles: dev
   datasource:
-    url: jdbc:h2:/tmp/{{project.name}}/sbr-gen-database.h2;DB_CLOSE_ON_EXIT=FALSE
+    url: jdbc:postgresql://localhost:5432/essDB
     username: easin
-    password:
-    driverClassName: org.h2.Driver
+    password: Easin
```

Comment:
let's remove Easin and use project name instead 
```suggestion
    password: {{project.name}-pwd!!
```